src/dvs2tensor.cpp

```diff
@@ -0,0 +1,296 @@
+#define LIBCAER_FRAMECPP_OPENCV_INSTALLED 0
+
+#include <libcaercpp/devices/davis.hpp>
+#include <libcaer/devices/davis.h>
+#include "dvs2tensor.hpp"
+
+#include <cstddef>
+#include <torch/csrc/autograd/python_variable.h>
+#include <torch/extension.h>
+#include <torch/script.h>
+#include <torch/torch.h>
+#include <type_traits>
+
+// ================================
+#include <atomic>
+#include <csignal>
```

Comment:
header ordering should be from own headers to third party libraries to standard library